pkg/api/run_now.go

```diff
@@ -0,0 +1,84 @@
+package api
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/flanksource/canary-checker/api/context"
+	"github.com/flanksource/canary-checker/checks"
+	"github.com/flanksource/canary-checker/pkg"
+	"github.com/flanksource/canary-checker/pkg/db"
+	"github.com/flanksource/commons/logger"
+	"github.com/labstack/echo/v4"
+	"gorm.io/gorm"
+)
+
+// RunNowRequest represents the request body for a run now request
+type RunNowRequest struct {
```

Comment:
Why can't we just use `/run/canary/{id}` and `/run/topology/{id}`

Reply:
Sure we can. The requirement wasn't clear whether we needed a single handler or 2 separate.